static/app/components/feedback/widget/feedbackWidget.tsx

```diff
@@ -17,14 +20,19 @@ export default function FeedbackWidget({
   title = 'Report a Bug',
   type,
 }: FeedbackWidgetProps) {
+  const config = useLegacyStore(ConfigStore);
+
   // Don't render anything if Sentry SDK is not already loaded
   if (!getCurrentHub()) {
     return null;
   }
 
+  const widgetTheme = config.theme === 'dark' ? 'dark' : 'light';
```

Comment:
We can't rely solely on the media query -- we need the ability to explicitly set the theme since that is how theming works on sentry (you can explicitly set 1) light, 2) dark, 3) system theme which would be affected by the media query)